onyo/lib/onyo.py

```diff
@@ -510,6 +512,32 @@ def _fsck_clean_tree(self) -> bool:
 
         return True
 
+    def _fsck_no_pseudo_keys(self) -> bool:
+        """
+        Check that no asset contains any pseudo-key names.
+        """
+        assets_with_pseudo_keys = {}
+        pseudo_keys = ["type", "make", "model", "serial"]
+
+        for asset in self.assets:
+            contents = self._read_asset(asset)
+
+            for p in pseudo_keys:
+                if p in contents:
+                    if asset not in assets_with_pseudo_keys.keys():
```

Comment:
No need for `.keys()` here. Simply: `if asset not in assets_with_pseudo_key:`

Nit: I find `assets_with_pseudo_keys.keys()` to be odd to read. Perhaps a more generic name, such as `assets_failed` or `assets_violation`?